app/actions/apps.ts

```diff
@@ -39,8 +43,9 @@ export function doAppCall<Res=unknown>(call: AppCall, intl: any): ActionFunc {
                     return {data: res};
                 }
 
-                // TODO Open new interactive dialog
-                //dispatch(openAppsModal(res.form, call));
+                if (call.type === AppCallTypes.SUBMIT) {
+                    showAppForm(res.form, call, getTheme(getState()));
```

Comment:
Self review: test and probably better handle form response to already opened form.